src/zope/traversing/namespace.py

```diff
@@ -11,7 +11,48 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""URL Namespace Implementations
+"""
+URL Namespace Implementations
+
+A URL Namespace is usually a path segment that looks like ``++ns++name``.
+(It can also look like ``@@name``, which is a
+shortcut for ``++view++name``. See :func:`nsParse` for details.)
```

Comment:
My inner pedant wants to say that namespace is *not* a path segment.  I'll try to suggest a rephrasing.  Hmm, maybe

> A namespace can be used in a path segment like ``++ns++name`` to disambiguate (e.g. items versus attributes), or access a different set of traversable names (e.g. views, accessible via ``++view++name``, or its special shortcut spelling ``@@name``).

Reply:
Good point, that was sloppy wording on my  part. How about:

> URL namespaces are an extensible mechanism to provide additional control over traversal (for example, disambiguating item versus attribute access) or access to an additional set of traversable names (such as :class:`registered views <view>` or :class:`path adapters
<adapter>`).  This mechanism is used for path segments that look like ``++ns++name``.  (It is also used for segments like ``@@name``, which is a shortcut for ``++view++name``.  See :func:`nsParse` for details.)